tests/integration/test_trading.py

```diff
@@ -294,3 +296,175 @@ def test_estimate_position(vega_service_with_market: VegaServiceNull):
         is_market_order=[False, False],
         collateral_available=1,
     )
+
+@pytest.mark.integration
+def test_liquidation_and_estimate_position(vega_service_with_market: VegaServiceNull):
```

Comment:
The `vega_service_with_market` fixture is going to create a market and as you are creating your own market with the same asset symbol and market name as the fixture it could cause some confusion in which one the test is actually using.

When creating a market within the test it may be better to use the `vega_service` fixture.